src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -9,91 +10,330 @@ import {
   SelfServeBaseClass,
   SmartUiInput,
 } from "../SelfServeTypes";
-import { refreshDedicatedGatewayProvisioning } from "./SqlX.rp";
+import { BladeType, generateBladeLink } from "../SelfServeUtils";
+import {
+  deleteDedicatedGatewayResource,
+  getCurrentProvisioningState,
+  refreshDedicatedGatewayProvisioning,
+  updateDedicatedGatewayResource,
+} from "./SqlX.rp";
+
+const costPerHourValue: Description = {
+  textTKey: "CostText",
+  type: DescriptionType.Text,
+  link: {
+    href: "https://azure.microsoft.com/en-us/pricing/details/cosmos-db/",
```

Comment:
i think there is aka ms link for this: [https://aka.ms/cosmosdbpricing](https://aka.ms/cosmosdbpricing).

Mostly, please stick to aka.ms link since it can be updated without having to redploy.

Reply:
Balaji mentioned that all these are mock links and that they will update it once the doc pages are available.